index.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
These files (index and the packaged tar.gz) don't go on the same branch as the code, please remove them from this branch and PR. Thanks!

Reply:
Why not? Particularly during the current rapid development cycle it gives one point of reference vs. having to maintain and fish for a chart repo?

Reply:
For development, you would rather use `helm install -f config.yaml ./galaxy-stable` locally, no need for packaging while developing. We put the packaged files and index in the github-io branch.

Reply:
That is the convention followed when hosting helm on github...

Reply:
This branch should be only for the chart logic, not for storing its packaged version. Thanks!

Reply:
OK, I had forgotten about the `gh-pages` branch. I'm using this primarily from Rancher, which requires a git repo. Instead of polluting that branch with repeated commits of the dev package then, I've just created a separate repo for the dev packages: https://github.com/CloudVE/helm-charts